spec/models/address_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+describe "Address model"  do
+  before {
+    @address = Address.new(short_address: "shortAddr1", full_address: "longAddr1",
+                     zipcode: 12345, country: "country1")
+    @address_short = Address.new(short_address: "shortAddr1", full_address: "longAddr2",
+                     zipcode: 12346, country: "country1")
+  }
```

Comment:
you did not test the associations.
structure should be:
describe associations
  describe ...
  describe ...
describe validation
  describe ...
  describe ...